src/components/DBSearchResults.vue

```diff
@@ -189,10 +189,10 @@
           <h3 class="mt-3 mb-2 fs-5">Cinematographer(s)</h3>
           <p class="m-3">{{getCrewMember(topStructure(result).crew, "Photo")}}</p>
 
-          <hr>
+          <hr v-if='turnArrayIntoList(topStructure(result).tags, "title")'>
 
-          <h3 class="mt-3 mb-2 fs-5">Tags</h3>
-          <p class="m-3">{{turnArrayIntoList(topStructure(result).tags, "title")}}</p>
+          <h3 class="mt-3 mb-2 fs-5" v-if='turnArrayIntoList(topStructure(result).tags, "title")'>Tags</h3>
+          <p class="m-3" v-if='turnArrayIntoList(topStructure(result).tags, "title")'>{{turnArrayIntoList(topStructure(result).tags, "title")}}</p>
```

Comment:
Since we're using this three times in a row here, maybe we should make a method called something like `hasTags` which could take a result and return `Boolean(this.turnArrayIntoList(topStructure(result).tags, "title"))`. That being said, this is fine as it is so you don't have to.

We need to hang out ona  call and do some PR work so we can figure out why you keep having to make new PRs.